tasks.py

```diff
@@ -175,6 +175,7 @@ def checkparts(c):
         "ad719x",
         "ad469x",
         "ad777x",
+        "ad7134",
```

Comment:
We only allow skipping this check in special cases. If you need help creating the context file please let us know

Reply:
Noted on this. I have removed AD7134 from this list. 
Is there a separate context file to be added while adding support for a new device? It would be really of help if someone could guide through this.

Reply:
Take a look here  https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#emulation
If you don't have a Linux box you can use conda to get iio-emu running:
```
git clone https://github.com/tfcollins/staged-recipes.git -b iio-emu
cd iio-emu/recipes
conda-build iio-emu
conda install --use-local iio-emu
```

Reply:
@tfcollins Few questions on this:
After I clone the staged-recipes.git, I do not see an **iio-emu/recipes** directory, instead I see a **recipes/iio-emu**.
If I execute the conda-build iio-emu inside the **recipes/iio-emu**  , then I see the following exception:

![image](https://user-images.githubusercontent.com/89130524/236108671-df632a74-f81b-4da8-9500-b49090b4ba8b.png)

Reply:
oops sorry typo. You'll probably need the conda forge channel as well:
```
git clone https://github.com/tfcollins/staged-recipes.git -b iio-emu
cd staged-recipes/recipes
conda config --add channels conda-forge
conda config --set channel_priority strict
conda-build iio-emu
conda install --use-local iio-emu
```

Reply:
Hi @tfcollins ,

The `conda-build iio-emu` gives an error saying "No valid recipes found for input"

![image](https://github.com/analogdevicesinc/pyadi-iio/assets/89130524/9a42e7b9-7179-4d6b-964f-e4adc84dc3f3)

Reply:
Please make sure you are using the iio-emu branch of staged-recipes

Reply:
Hi @tfcollins I am too facing similar problem while building iio_emu. Is there any document guide for the same?
Also is this used to generate the device.xml file? I was looking at other device xml files and they do have entire IIO xml context info including attributes, channels info, scan structure, etc. So was wondering from where is this information is coming from. Is it coming from Linux drivers? If yes, how do we generate one for the devices that don't have linux drivers support yet?
e.g. https://github.com/analogdevicesinc/pyadi-iio/blob/master/test/emu/devices/ad7768.xml

Reply:
These context files come from the hardware itself. Its not possible to make them without a running board + driver

Reply:
Ok, so is there a guide for creating this file? Is this file created automatically or manually from the IIO context information?
We don't really work on the Linux drivers much, so while supporting these drivers can we differ the iio_emu emulation context step?